lib/model/store.dart

```diff
@@ -364,6 +372,7 @@ class PerAccountStore extends ChangeNotifier with EmojiStore, ChannelStore, Mess
   final Map<int, User> users;
 
   final TypingStatus typingStatus;
+  final TypingNotifier typingNotifier;
```

Comment:
Yeah, `TypingNotifier` itself only maintains its internal states, so I wasn't sure if `PerAccountStore` is the best fit. Moving it to `UpdateMachine` sounds like a creative idea, but I'm not sure do we typically want to access it from our widget code.

Reply:
I think of `UpdateMachine` as mainly having the job of keeping its associated `PerAccountStore` up to date with data from the server.

Then with `registerNotificationToken` it got a bit of mission creep. What really ties that method to the rest of `UpdateMachine` is: it's code that in the live app is associated one-to-one with a `PerAccountStore`, but that on its own initiative goes and makes HTTP requests to the server, and that therefore we don't want to be active in our tests except where specifically relevant. (Because if it were active, it'd be a pain to manage those HTTP requests interleaving with whatever other HTTP requests any particular test is interested in.) There's probably a better way to organize that but that's how it currently is.

With `TypingNotifier`, there's no need to take any particular steps to suppress it in the bulk of our tests — it only does anything when its methods get called. The more narrowly-targeted, and explicit, suppression that a handful of files end up needing in this PR works fine — and the explicitness is nice. So `PerAccountStore` is a good home.